### PR TITLE
Updated user redux and folder structure

### PR DESCRIPTION
**Pull request by DenEdwards**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,28 +5,28 @@ import {
...
 import Register from "./pages/Register";
 import Home from "./pages/Home";
 import Profile from "./pages/Profile";
 import Settings from "./pages/Settings";
 import Datapage from "./pages/Datapage";
-import BlockedLogin from "./components/BlockedLogin";
+import BlockedLogin from "./components/generic/BlockedLogin";
 import KabinetDashboard from "./pages/KabinetDashboard";
-import Page from "./components/Page";
+import Page from "./components/Kabinet/Page";
```
Comment (vladislab) [inline src/App.js]: could you revert this

File: src/components/Kabinet/Page.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import "./styles/Page.scss";
+import "../styles/Page.scss";
```
Comment (vladislab) [inline src/components/Kabinet/Page.js]: and this

File: src/pages/KabinetDashboard.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { Grid, Button, TextField, Paper, Typography } from "@material-ui/core";
 import "./styles/KabinetDashboard.scss";
 import { DatePicker } from "@material-ui/pickers";
-import IdeaCard from "../components/IdeaCard";
+import IdeaCard from "../components/Kabinet/IdeaCard";
```
Comment (vladislab) [inline src/pages/KabinetDashboard.js]: this too

File: src/components/Kabinet/IdeaCard.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ import {
   MenuItem,
 } from "@material-ui/core";
 import { MoreVert } from "@material-ui/icons";
-import empty from "../assets/empty-state-photo.png";
+import empty from "../../assets/empty-state-photo.png";
```
Comment (vladislab) [inline src/components/Kabinet/IdeaCard.js]: and this

Comment (DenEdwards) [reply src/components/Kabinet/IdeaCard.js]: Yeah are we splitting it up later?

**Review by vladislab — CHANGES_REQUESTED** [review]
can you revert the highlighted part ?
Ill move those stuff myself so my current commit wont conflict

**Review by vladislab — APPROVED** [review]

**Pull request merged**